engines/kyra/engine/chargen.cpp

```diff
@@ -1235,10 +1307,16 @@ int CharacterGenerator::modifyStat(int index, int8 *stat1, int8 *stat2) {
 	printStats(_activeBox, 3);
 	_vm->removeInputTop();
 
+	Screen::FontId of = _screen->setFont(Screen::FID_8_FNT);
```

Comment:
This is the wrong font for the for the PC-98 games.
I have checked which one of the presets we can use: It seems that `Screen::FontId of = _screen->setFont(_vm->_invFont3);` will be correct for all games (Chinese, other DOS/Amiga, FM-Towns, PC-98 and SegaCD).

Reply:
Moved this code to Chinese block. Is it ok with you

Reply:
Sure, that's fine.